pkg/virt-launcher/virtwrap/converter/network.go

```diff
@@ -87,7 +87,8 @@ func CreateDomainInterfaces(vmi *v1.VirtualMachineInstance, c *ConverterContext)
 			domainIface.Type = "ethernet"
 			if iface.BootOrder != nil {
 				domainIface.BootOrder = &api.BootOrder{Order: *iface.BootOrder}
-			} else {
+			} else if !isS390X(vmi.Spec.Architecture) {
```

Comment:
As follow up PR, this should probably go into the archconverter method

Reply:
I think that would make sense to be done in the same PR as discussed here: https://github.com/kubevirt/kubevirt/pull/10490#discussion_r1632782955